pkg/controller/common/utils.go

```diff
@@ -0,0 +1,54 @@
+package common
```

Comment:
I wasn't sure if I could add the `GetGitSecret()` to `git/secret.go` because I had a feeling that the `git` package should stay as kind of framework without any dependencies to k8s API. But I'm fine with moving it there. And the rest....well what I can say - just that it was too late and I was running in the safe mode :-). But that is no excuse for me ;-)
refactored in 0020aa9